### PR TITLE
Terraform docs to inject generated content into one section of README 

### PR DESCRIPTION
**Pull request by rarsan**
This is done to replace only relevant Terraform inputs/outputs content in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<!-- BEGIN_TF_DOCS -->
```
Comment (Lakhtenkov-iv) [inline README.md:1]: Please remove it in the end.
https://github.com/GoogleCloudPlatform/terraform-splunk-log-export/pull/24/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R152

Comment (rarsan) [reply README.md:1]: Wouldn't that fail the doc validation check?

Comment (Lakhtenkov-iv) [reply README.md:1]: We can give it a try, at least generation locally works fine without it. But I had no chance to run GH action

Comment (rarsan) [reply README.md:1]: Ah good catch. I thought you were referring to the other one. Didn't notice there was another closing comment at the end. Fixed.

**Pull request merged**